lesson-7/server/api.js

```diff
@@ -0,0 +1,53 @@
+const MongoClient = require('mongodb').MongoClient;
+
+let database;
+let collectionName;
```

Comment:
Don't do global variables. It will rewrite if you attempt to use this library to get two connections. It may not be a problem in this lesson but it is a bad example.